res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
Legally it is allowed to move from GPLv2 or later to GPLv2 only - but that loses the ability to reuse code under Apache.

Also I would strongly object merging any pull-request which does that. It would break possible securities for users who rely on strong copyleft for protection of their freedoms - like being able to test whether what they get was tampered with.